src/PrestaShopBundle/Resources/views/Admin/Multistore/header.html.twig

```diff
@@ -86,31 +87,31 @@
         <div class="multishop-modal-dialog" role="document">
           <div class="multishop-modal-body">
             {% if lockedToAllShopContext == false %}
-            <div class="multishop-modal-search-container">
-              <i class="material-icons">search</i>
-              <input type="text" class="form-control multishop-modal-search js-multishop-modal-search" placeholder="Search shop name" data-no-results="{{ "No results found for"|trans({}, 'Admin.Global') }}" data-searching="{{ "Searching for"|trans({}, 'Admin.Global') }}">
-            </div>
+              <div class="multishop-modal-search-container">
+                <i class="material-icons">search</i>
+                <input type="text" class="form-control multishop-modal-search js-multishop-modal-search" placeholder="Search shop name" data-no-results="{{ "No results found for"|trans({}, 'Admin.Global') }}" data-searching="{{ "Searching for"|trans({}, 'Admin.Global') }}">
+              </div>
             {% endif %}
 
             <ul class="multishop-modal-group-list js-multishop-scrollbar">
-                <li class="multishop-modal-all multishop-modal-item">
-                  {% if isAllShopContext %}
-                    <i class="material-icons">check</i>
-                  {% else %}
-                    <span class="multishop-modal-color" style="background-color:#25b9d7;"></span>
-                  {% endif %}
-                  <a class="multishop-modal-all-name" href="{{ app.request.requestURI ~ '&setShopContext=' }}">
-                    <span>{{ "All shops"|trans({}, 'Admin.Global') }}</span>
-                  </a>
-                </li>
+              <li class="multishop-modal-all multishop-modal-item">
+                {% if isAllShopContext %}
+                  <i class="material-icons">check</i>
+                {% else %}
+                  <span class="multishop-modal-color" style="background-color:#25b9d7;"></span>
+                {% endif %}
+                <a class="multishop-modal-all-name" href="{{ app.request.requestURI ~ '&setShopContext=' }}">
```

Comment:
Is there something missing after `&setShopContext=` ?

Reply:
yes it works like this...  but all this HTML part doesn't come from this PR, it's just that my code editor changed the indentation :D